internal/gps/result.go

```diff
@@ -42,37 +45,61 @@ type solution struct {
 	solv Solver
 }
 
-// WriteDepTree takes a basedir and a Lock, and exports all the projects
-// listed in the lock to the appropriate target location within the basedir.
+// WriteDepTree takes a baseDir and a Lock, and exports all the projects
+// listed in the lock to the appropriate target location within baseDir.
 //
-// If the goal is to populate a vendor directory, basedir should be the absolute
+// If the goal is to populate a vendor directory, baseDir should be the absolute
 // path to that vendor directory, not its parent (a project root, typically).
 //
-// It requires a SourceManager to do the work, and takes a flag indicating
-// whether or not to strip vendor directories contained in the exported
-// dependencies.
-func WriteDepTree(basedir string, l Lock, sm SourceManager, sv bool) error {
+// It requires a SourceManager to do the work, and takes a PruneOptions
+// indicating the pruning options required for the exported dependencies.
+func WriteDepTree(baseDir string, l Lock, sm SourceManager, prune PruneOptions, logger *log.Logger) error {
```

Comment:
Oops, this looks like unmerged commits from #968 snuck in? Or did you intend to combine these PRs?